src/js/index.js

```diff
@@ -20,16 +20,39 @@ const filterInput = document.querySelector('#filter');
 const sortingInput = document.querySelector('#sorting');
 const tagsSelect = document.querySelector('#tags');
 const showExpired = document.querySelector('#expired');
+const hideCompleted = document.querySelector('#completed');
 
 const activateElements = els => Array.from(els).forEach(node => node.classList.add(ACTIVE_CLASS));
 const allowDifficultySelect = shouldAllow => sortingInput.querySelectorAll('.difficulty')
 	.forEach(node => {
 		node.disabled = !shouldAllow;
 	});
+const getCompletedItems = () => JSON.parse(localStorage.getItem('devswag:completed')) || [];
 
 let search;
 let selectr;
 
+function setCompleteAttr(el) {
+	Array.from(document.querySelectorAll('.item'))
+		.find(item => item.contains(el)).dataset.completed = el.checked;
```

Comment:
I think using a classname would be better here - `.classList.toggle('checked-off', el.checked)`